app/assets/stylesheets/app/documents.css.scss

```diff
@@ -1,4 +1,4 @@
-tr.hidden-folder {
+tr.hidden-folder, tr.hidden-document {
```

Comment:
Avoid qualifying class selectors with an element.<br>Each selector in a comma sequence should be on its own line